third-party/realdds/include/realdds/dds-participant.h

```diff
@@ -38,11 +38,24 @@ class dds_participant
     eprosima::fastdds::dds::DomainParticipant * get() const { return _participant; }
     eprosima::fastdds::dds::DomainParticipant * operator->() const { return get(); }
 
-    // Every participant in DDS has a global identifier, comprising a prefix- and entity-ID.
-    // The prefix (vendor, host, process, participant) usually does not vary within the same participant.
+    // RTPS 8.2.4.2 "Every Participant has GUID <prefix, ENTITYID_PARTICIPANT>, where the constant ENTITYID_PARTICIPANT
```

Comment:
Good information.
Won't `dds-guid.h` be a better place for it. Or next to `dds_guid` definition in `dds-defines.h`?

Reply:
Hmm not sure, but this particular information is specific to participant GUIDs.